tests/test_worker_pool.py

```diff
@@ -84,3 +84,41 @@ async def test_exit_respawn(worker_pool):
     assert len(exceptions) == worker_pool.workers * 3
     for exc in exceptions:
         assert isinstance(exc, ProcessError)
+
+
+INITIALIZER_ARGS = None
+INITIALIZER_KWARGS = None
+
+
+def initializer(*args, **kwargs):
+    global INITIALIZER_ARGS, INITIALIZER_KWARGS
+    INITIALIZER_ARGS = args
+    INITIALIZER_KWARGS = kwargs
+
+
+def get_initializer_args():
+    return INITIALIZER_ARGS, INITIALIZER_KWARGS
```

Comment:
Use closure instead of global variables

Reply:
It's will be returned from separate process, closure useless.